app/src/main/java/com/it/hungvt/movieapp/utils/Constant.java

```diff
@@ -0,0 +1,18 @@
+package com.it.hungvt.movieapp.utils;
+
+/* Define constants variable */
+
+import android.support.annotation.IntDef;
```

Comment:
ERROR: Unused import: android.support.annotation.IntDef. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports